src/devices/systemReady/SystemReady_nws_yarp.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-FileCopyrightText: 2006-2021 Istituto Italiano di Tecnologia (IIT)
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef YARP_SYSTEMREADY_NWS_YARP_H
+#define YARP_SYSTEMREADY_NWS_YARP_H
+
+#include <yarp/sig/Vector.h>
```

Comment:
This is not used, but two other includes are missing instead: `<vector>` and `<yarp/os/Port.h>`.